app/controllers/claims/schools/claims/mentors_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Claims::Schools::Claims::MentorsController < ApplicationController
```

Comment:
```suggestion
class Claims::Schools::Claims::MentorsController < Claims::ApplicationController
```
Since https://github.com/DFE-Digital/itt-mentor-services/pull/308/files when in you'll need to change the controller.
You will also likely need to add policies.

Reply:
https://github.com/DFE-Digital/itt-mentor-services/pull/307/commits/bdf5eeb47d67673f54f86da9385cae81f5796f6a